configure.ac

```diff
@@ -325,6 +325,42 @@ fi
 AC_SUBST([LAZPERF_STATUS])
 AC_SUBST([LAZPERF_CPPFLAGS])
 
+
+dnl ===========================================================================
+dnl Figure out if we are postgres >= 9.6 and support PARALLEL
+
+
+dnl Extract the version information from pg_config
+dnl Note: we extract the major & minor separately, ensure they are numeric, and then combine to give
+dnl the final version. This is to guard against user error...
+PGSQL_FULL_VERSION=`"$PG_CONFIG" --version`
+
+PGSQL_MAJOR_VERSION=`echo $PGSQL_FULL_VERSION | sed 's/[[^0-9]]*\([[0-9]]*\).*/\1/'`
+PGSQL_MINOR_VERSION=`$PG_CONFIG --version | sed 's/[[^0-9]]*\([[0-9]]\)\.\([[0-9]]\).*/\2/'`
+
+if test $PGSQL_MAJOR_VERSION -gt 9; then
+dnl ==================================================================
+dnl Starting with PostgreSQL 10, major is the new minor
+dnl This is to prevent things like 10.31 ranking higher than 11.0
+dnl===================================================================
+PGSQL_MINOR_VERSION=0
+fi
+
+
+AC_MSG_RESULT([checking PostgreSQL version... $PGSQL_FULL_VERSION])
+
+POSTGIS_PGSQL_VERSION="$PGSQL_MAJOR_VERSION$PGSQL_MINOR_VERSION"
```

Comment:
PostGIS [uses](https://github.com/postgis/postgis/blob/68fc1fec36dc48ac507dc4c8ee3e3f102a68243a/configure.ac#L453) the `POSTGIS_` prefix because it's PostGIS :) Let's not use the `POSTGIS_` prefix in the Pointcloud extension.